### PR TITLE
Add the option to add more than 1 firewall rule

### PR DESCRIPTION
**Pull request by cregev**
This will add support for hiera too.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-firewall",
-  "version": "1.8.0",
+  "version": "1.8.1",
```
Comment (hunner) [inline metadata.json]: Please don't bump the metadata as part of a PR; we do that separately.

File: metadata.json [hunk 2]
```diff
@@ -80,6 +80,6 @@
     }
   ],
   "dependencies": [
-  
+      {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 <5.0.0"}
```
Comment (hunner) [inline metadata.json]: Adding a dependency is considered a backwards-incompatible change normally I think, though basically everyone has stdlib installed...

File: manifests/init.pp [hunk 3]
```diff
@@ -14,6 +14,7 @@
 class firewall (
   $ensure       = running,
   $pkg_ensure   = present,
+  $rules        = {},
```
Comment (hunner) [inline manifests/init.pp]: New parameters would need a documentation update.

**Comment by jonnytpuppet** [comment]
@cregev I'm sorry I don't believe this is the neatest way to solve this problem. What we have done in other modules, is create a separate class for supporting heira. Perhaps you would consider doing the same, rather than adding to the main firewall class? Also, we need a docs update. Thanks!

**Comment by jonnytpuppet** [comment]
Hi There

Thanks for your efforts!

I'm closing this PR due to inactivity. If you would still like to see this feature, feel free to open a new PR with the comments addressed.

Thanks

**Pull request closed without merging**